### PR TITLE
search sidebar overscroll

### PR DESCRIPTION
**Pull request by KarinaMendez2**

### CODE REVIEW

**Review by TomWoodward — DISMISSED** [review]

File: src/app/content/search/components/SearchResultsSidebar/SearchResultsBarWrapper.tsx [hunk 1]
```diff
@@ -118,6 +119,8 @@ export class SearchResultsBarWrapper extends Component<ResultsSidebarProps> {
     const {callback, deregister} = setSidebarHeight(searchSidebar, window);
     callback();
     this.deregister = deregister;
+
+    searchSidebar.addEventListener('webkitAnimationEnd', fixSafariScrolling);
```
Comment (TomWoodward) [inline src/app/content/search/components/SearchResultsSidebar/SearchResultsBarWrapper.tsx]: you need to `removeEventListener` in `componentWillUnmount` too

File: src/app/content/search/components/SearchResultsSidebar/index.spec.tsx [hunk 2]
```diff
@@ -23,21 +24,29 @@ import {
...
 } from '../../actions';
+import { SearchResultsBarWrapper } from './SearchResultsBarWrapper';
 
 describe('SearchResultsSidebar', () => {
   let store: Store;
   let dispatch: jest.SpyInstance;
 
+  const animationEvent = () => {
+    const event = new (assertWindow().CustomEvent)('webkitAnimationEnd');
+    event.initEvent('webkitAnimationEnd', true, true);
```
Comment (TomWoodward) [inline src/app/content/search/components/SearchResultsSidebar/index.spec.tsx]: did you try removing this? not sure if its required

File: src/app/content/search/components/SearchResultsSidebar/index.spec.tsx [hunk 3]
```diff
@@ -175,4 +184,24 @@ describe('SearchResultsSidebar', () => {
...
+    const {tree} = renderToDom(render());
+    const fixForSafariMock = jest.spyOn(domUtils, 'fixSafariScrolling');
+
+    store.dispatch(requestSearch('cool search'));
+    store.dispatch(receiveSearchResults(makeSearchResults()));
+
+    const sidebar = ReactTestUtils.findRenderedComponentWithType(tree, SearchResultsBarWrapper);
+
+    jest.useFakeTimers();
+    setTimeout(() => {
```
Comment (TomWoodward) [inline src/app/content/search/components/SearchResultsSidebar/index.spec.tsx]: i don't think you need this `setTimeout`

File: src/app/theme.ts [hunk 4]
```diff
@@ -102,6 +102,7 @@ export default {
     modal: 12,
     navbar: 5,
     overlay: 3,
+    searchSidebar: 1,
```
Comment (TomWoodward) [inline src/app/theme.ts]: we want this to be above `contentNotifications`

**Review by TomWoodward — CHANGES_REQUESTED** [review]

File: src/app/content/search/components/SearchResultsSidebar/index.spec.tsx (hunk 3 above)
Comment (KarinaMendez2) [reply src/app/content/search/components/SearchResultsSidebar/index.spec.tsx]: I tried removing it. Didn't work, don't really know what you mean.

File: src/app/theme.ts (hunk 4 above)
Comment (KarinaMendez2) [reply src/app/theme.ts]: like the variable? or the actual index number?

Comment (TomWoodward) [reply src/app/theme.ts]: the index number. with what you have here, if you trigger the updates available it'll show up on top of the sidebar

**Review by TomWoodward — APPROVED** [review]

**Pull request merged**